paparazzi/src/main/java/app/cash/paparazzi/deprecated/com/android/ide/common/resources/deprecated/MultiResourceFile.java

```diff
@@ -166,7 +171,10 @@ private void updateResourceItems(ScanningContext context) {
     private void parseFile() {
         try {
             SAXParser parser = XmlUtils.createSaxParser(sParserFactory);
-            parser.parse(getFile().getContents(), new ValueResourceParser(this, isFramework(), null));
+            InputStreamReader reader = new InputStreamReader(getFile().getContents());
+            InputSource source = new InputSource(reader);
```

Comment:
[InputSource.setEncoding](https://docs.oracle.com/javase/8/docs/api/org/xml/sax/InputSource.html#setEncoding-java.lang.String-):

> This method has no effect when the application provides a character stream.

A `Reader` is a "character stream", so I think this (and previous line) can be abbreviated to:

```suggestion
            InputSource source = new InputSource(getFile().getContents());
```
and this also makes the encoding change (next line) take effect according to docs.